.github/workflows/clang-linux-nix-check.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           nix build -L .?#checks.x86_64-linux.crypto3-clang
         env:
           NIX_CONFIG: |
-            cores = 4
+            cores = 16
```

Comment:
I see you were able to increase the number of cores. Did the memory usage decline after pre-compiled headers, and that allowed you to use more cores?

Reply:
Yes, exactly!